src/components/NavBar.js

```diff
@@ -27,19 +27,19 @@ const NavBar = ({ history, authToken, logOut }) => {
       </Button>
     </Link>
   ) : (
-      <React.Fragment key="group">
-        <SearchBarAutoComplete key="searchBarAutoComplete"/>
-        <Link to="/" key="link-signout">
-          <Button
-            key="1"
-            style={{ marginTop: '15px', color: '#549bea' }}
-            onClick={logOut}
-          >
-            Sign Out
-            <LogoutOutlined />
-          </Button>
-        </Link>
-      </React.Fragment>
+    <React.Fragment key="group">
```

Comment:
I could be wrong on this,but I don't believe fragments are rendered so shouldn't need a `key` prop. Also, you're using the short hand for fragments in other parts of your code. I'd chose the shorthand as it's the convention throughout the app. If you feel that's hard to read then we can all have a conversation about writing `<React.Fragment>` so that the code base is consistent.

Reply:
That's actually something I implemented in #58 to address a warning being thrown by React.
https://github.com/Lambda-School-Labs/frontend-vbb-portal/pull/58#discussion_r638630891